ansible/roles/hyrax/tasks/compel.yml

```diff
@@ -0,0 +1,44 @@
+---
+- block:
+  - name: load workflow
+    command: ./bin/rails hyrax:workflow:load
+    args:
+      chdir: '{{ project_app_root }}'
+
+  - name: create default collection type
+    command: ./bin/rails compel:create_default_collection_type
```

Comment:
Stylistically, `bundle exec` is preferred over `./bin/`.  E.g., `bundle exec rails ...` rather than `./bin/rails ...`.